sessionctx/stmtctx/stmtctx.go

```diff
@@ -340,6 +340,8 @@ type StatementContext struct {
 		HasFKCascades bool
 	}
 
+	// TableStats stores the visited runtime table stats by table id during query
+	TableStats map[int64]interface{}
```

Comment:
`interface{}` seems not clear enough, why not use `*statistics.Table`?

Reply:
because use `*statistics.Table` would introduce cycle import